static/app/views/issueDetails/header.tsx

```diff
@@ -83,6 +84,12 @@ function GroupHeaderTabs({
     group_has_replay: (replaysCount ?? 0) > 0,
   });
 
+  useEffect(() => {
+    if (organization && group.issueType === IssueType.REPLAY_HYDRATION_ERROR) {
+      trackAnalytics('replay.hydration-error.issue-details', {organization});
+    }
```

Comment:
typescript thinks that organization will always be set here, so we should be good without it in the check
```suggestion
    if (group.issueType === IssueType.REPLAY_HYDRATION_ERROR) {
      trackAnalytics('replay.hydration-error.issue-details', {organization});
    }
```